src/components/progress-tracker/progress-tracker.test.js

```diff
@@ -169,4 +181,135 @@ context("Tests for ProgressTracker component", () => {
       }
     );
   });
+
+  describe("should checkt accessibility tests for progress tracker component", () => {
+    describe.each([
+      "aria-label",
+      "aria-describedby",
+      "aria-valuenow",
+      "aria-valuemin",
+      "aria-valuemax",
+      "aria-valuetext",
+    ])("when %s prop is passed", (propName) => {
+      it(`check the accessibility when the ${propName} is set to cypress-standard`, () => {
+        let props;
+        if (
+          !propName.endsWith("now") &&
+          !propName.endsWith("min") &&
+          !propName.endsWith("max")
+        ) {
+          props = { [propName]: CHARACTERS.STANDARD };
+        } else {
+          props = { [propName]: 50 };
+        }
+
+        CypressMountWithProviders(<ProgressTrackerComponent {...props} />);
+
+        cy.checkAccessibility();
+      });
+    });
+
+    it.each([
+      PROGRESS_TRACKER_SIZES[0],
+      PROGRESS_TRACKER_SIZES[1],
+      PROGRESS_TRACKER_SIZES[2],
+    ])(
+      "should check the accessibility when component is rendered with %s size",
+      (size) => {
+        CypressMountWithProviders(<ProgressTrackerComponent size={size} />);
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each([150, 350, 550])(
```

Comment:
Could you not just pass these as strings with `px` and update the test description to be:
```
it.each(["150px", "350px", "550px"])(
"should check the accessibility when component is rendered %s length",
(length) => {
        CypressMountWithProviders(
          <ProgressTrackerComponent length={length} />
        );
```